src/pretix/plugins/checkinlists/exporters.py

```diff
@@ -240,7 +252,7 @@ def get_story(self, doc, form_data):
 
             row = [
                 '!!' if op.item.checkin_attention or op.order.checkin_attention else '',
-                CBFlowable(bool(op.last_checked_in)),
+                CBFlowable(checked=bool(op.last_checked_in), auto_checked_in=bool(op.auto_checked_in)),
```

Comment:
I don't think we *really* need to make this PDF more crowded by this information, I'm not sure in what use case it would be relevant.